docs/releases/5.2.txt

```diff
@@ -226,6 +226,9 @@ Models
 Requests and Responses
 ~~~~~~~~~~~~~~~~~~~~~~
 
+* Added a cached ``text`` property to :class:`~django.http.HttpResponse` for
+  decoding content.
```

Comment:
```suggestion
* The new :attr:`.HttpResponse.text` property decodes
  :attr:`.HttpResponse.content`.
```
This should be in present tense and we're missing docs for the new property (which we could link to here)
Something like...
```
.. attribute:: HttpResponse.text

    .. versionadded:: 5.2

    A decoded string representation of :attr:`HttpResponse.content`.

```
Under `.. attribute:: HttpResponse.content` maybe

Reply:
Thank you! I've made the changes as you suggested!

Reply:
A minor nitpick comment, I don't think it's necessary to say a "decoded" string, just saying "string" means is a `str` and not bytes.